factory.js

```diff
@@ -376,7 +376,7 @@ Factory.prototype.files = function (fn) {
         return next(null, {
           content: fullPath,
           compressed: fullPath + '.gz',
-          fingerprint: src.fingerprint || fingerprinter(factory.entry, { content: src }).id,
+          fingerprint: src.fingerprint || fingerprinter(fullPath, { content: src.content || src }).id,
```

Comment:
what causes content to be available under `src.content` rather than just `src`?

Reply:
@Swaagie specifically minify as we start to have more metadata to keep track of so we can no longer just have a filename as key with the content as value